homeassistant/config.py

```diff
@@ -130,7 +130,7 @@
 
 PACKAGES_CONFIG_SCHEMA = vol.Schema({
     cv.slug: vol.Schema(  # Package names are slugs
-        {cv.slug: vol.Any(dict, list, None)})  # Only slugs for component names
+        {cv.string: vol.Any(dict, list, None)})  # Component names are strings
```

Comment:
Since not all strings are valid component names, I don't think this comment is the best (even if it is technically correct).

Probably "Component configuration" would be enough of a comment to know what the line is for. The string type can be read easily in the actual code.

Reply:
I'm not sure `cv.string` is quite the right choice here either, but I couldn't find anything that seemed more right...

Reply:
Ok. I've been looking a bit at the code, and the config schema for packages seems to be much stricter than that for e.g. `configuration.yaml` - which is put into the pyyaml parser as is.

I see two options

- Leave it as `cv.string`, which puts it at the same level of checks as `configuration.yaml`, which is arguably not enough - doesn't feel quite _right_.
- Add a new validator that checks for a slug possibly followed by a space and a string - feels like adding a lot of code clutter for minimal gain.